Control coverage/Feature/VirtualMachineScaleSet.md

```diff
@@ -45,10 +46,10 @@ Diagnostics logs are needed for creating activity trail while investigating an i
 ### Control Spec
 
 > **Passed:**
-> Required diagnostics extension is present in VM Scale Set.
+> Required diagnostic extension is present in VM Scale Set.
```

Comment:
in vmss it is called as diagnostics extension only.

Reply:
Fixed.